File: src/main/java/com/bookmakerApp/model/EventModel.java
package com.bookmakerApp.model;

import com.bookmakerApp.model.enums.ChosenResult;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import javax.validation.constraints.DecimalMax;
import javax.validation.constraints.DecimalMin;
import java.util.Date;

@Getter
@Setter
@Entity
public class EventModel {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long idEvent;
    @DecimalMin("1.01")
    @DecimalMax("10.00")
    private Double odds;
    private Boolean success;
    private boolean finish;
    private Date date;
    @OneToOne
    @JoinColumn(name = "idSport")
    private SportModel sport;
    @Enumerated(EnumType.STRING)
    private ChosenResult chosenResult;
    private boolean resultIsChecked;
}

File: src/main/java/com/bookmakerApp/view/BookmakerClientApp/BookmakerClientApp/Data/Model/Event/EventModel.cs
using System;
using Newtonsoft.Json;

namespace BookmakerClientApp.Data.Model.Event;

public class EventModel
{
    [JsonProperty(PropertyName = "idEvent")]
    private long idEvent;
    private string chosenResult;
    private double odds;
    private DateTime date;
    private bool? success;
    private int numberOfPages;
    
    public long IdEvent { get => idEvent; set => idEvent = value; }
    public string ChosenResult { get => chosenResult; set => chosenResult = value; }
    public double Odds { get => odds; set => odds = value; }
    public DateTime Date { get => date; set => date = value; }
    public bool? Success { get => success; set => success = value; }
    public int NumberOfPages { get => numberOfPages; set => numberOfPages = value; }
}